tests/bootstrap.php

```
@@ -0,0 +1,914 @@
+        }
+        return date($format, (int)$time);
+    }
+}
+
+/**
+ * Stub XoopsSecurity for tests — provides createToken() without sessions.
+ */
+if (!class_exists('XoopsSecurity')) {
+    class XoopsSecurity
+    {
+        public function createToken($timeout = 0, $name = 'XOOPS_TOKEN')
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$timeout'. (undefined)

(UnusedFormalParameter)